src/GrpcProtos/capture.proto

```diff
@@ -27,6 +27,14 @@ message InstrumentedFunction {
   string function_name = 5;
 }
 
+message ApiFunction {
```

Comment:
I still feel like a small documentation comment here would help, even if the name changed, because the purpose is quite obscure unless one has reviewed this change. It can also point you to `FindApiFunctions`.

Reply:
Added some doc.